Alicerce/Sources/Logging/Log+NodeProvider.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Log+NodeProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class NodeProvider : LogProvider {
+
+        private static let dispatchQueueLabel = "Alicerce-Log"
+        private static let defaultRequestTimeout: TimeInterval = 0
+
+        private let serverURL: URL
+        private let operationQueue = OperationQueue()
+        private let requestTimeout: TimeInterval
+
+        public var minLevel: Log.Level = .error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
```

Comment:
While generally makes sense, in this particular case ```ConsoleProvider``` wouldn't conform to the ```LogProviderprotocol``` anymore.